include/fizzy/fizzy.h

```diff
@@ -410,6 +410,7 @@ bool fizzy_module_has_start_function(const FizzyModule* module) FIZZY_NOEXCEPT;
 /// @param  imported_globals           Pointer to the imported globals array. Can be NULL iff
 ///                                    @p imported_globals_size equals 0.
 /// @param  imported_globals_size      Size of the imported global array. Can be zero.
+/// @param  memory_pages_limit         Hard limit for memory growth in pages. Cannot be above 65536.
```

Comment:
Hmm, would it be a good idea allowing `0` for allowing "default setting"? Or perhaps someone actually wants to disable memory, and then `0` makes sense for that? If so, perhaps `uint32_t(-1)` to signal "use default"?

Reply:
I'm not sure, not a fan of special magic values.

Reply:
What is annoying is hardcoding 4096 in multiple places, like in rust.

Reply:
We could define default constant in `fizzy.h`, then it could be reused in tests and in rust.

Reply:
Yeah I think we should export the "default".

Reply:
I agree here. Looks good otherwise.